documentation/specs/Card.md

```diff
@@ -35,54 +35,44 @@ Architecture proposed is to surface a basic `<Card />` component with `variety`
 ### API
 
 ```ts
-export type CardAs = "div" | "label" | "button" | "a" | "fieldset";
-export type CardBackground = "primary" | "secondary";
-export type CardStatus = "danger" | "warning" | "success";
-export type CardVariant = "solid" | "outlined" | "flagged";
+type CardBackground = "primary" | "secondary";
+type CardVariant = "solid" | "outlined" | "flagged";
+type CardStatus = "danger" | "warning" | "success";
 
-type BaseCardContainerProps = {
+type CardContainerProps = {
   /** Custom element for the card container. */
-  as?: CardAs;
+  as?: ElementType;
 
-  /** Content of the card container. */
+  /** Content of the card. */
   children: ReactNode;
 
-  /** Render the card as disabled. */
+  /** Render the card as disabled. Only relevant for outlined cards. */
   isDisabled?: boolean;
 
-  /** Render the card as selected. */
+  /** Render the card as selected. Only relevant for outlined cards. */
   isSelected?: boolean;
 
+  /**
+   * Card status. Only relevant for flagged cars.
+   */
+  status?: CardStatus;
+
   /**
    * Card variant.
    * @default solid
```

Comment:
**Question:** 
Is this correct? In the `.mdx` file it says:
> By default cards are rendered with a border. This is the same as `variant="outlined"`.

Reply:
good catch